app/policies/service_provider_mfa_policy.rb

```diff
@@ -1,5 +1,5 @@
 class ServiceProviderMfaPolicy
-  AAL3_METHODS = %w[webauthn piv_cac].freeze
+  AAL3_METHODS = %w[webauthn webauthn_platform piv_cac].freeze
```

Comment:
What does this change do?

Reply:
When a service provider requests a phishing-resistant authenticator (here referred to as "AAL3"), these are the methods that are allowed.  Right now, both roaming and platform webauthn authenticators are represented by the `'webauthn'` value, but it's worth distinguishing them (we'll need to change it [here](https://github.com/18F/identity-idp/pull/6824#discussion_r952860114))